src/main/java/app/studnicki/ox/Main.java

```diff
@@ -9,6 +9,17 @@
  * Because of that there is ability to play after args being passed.
  */
 class Main {
+
+  private static final int FIRST_PLAYER_NAME_FIELD = 0;
```

Comment:
Ugh. Jak już masz 6 pól to to powinna być wyliczanka.

Reply:
Poprawione.